lib/CachePlugin.js

```diff
@@ -6,18 +6,37 @@
 
 const asyncLib = require("neo-async");
 
+/**
+ * Adds a cache to the compiler, where modules are cached.
+ * You can pass a cache object, where the modules are cached. Otherwise one is created per plugin instance.
+ */
 class CachePlugin {
+	/**
+	 *
+	 * @param {Object=} cache cache object
+	 */
 	constructor(cache) {
 		this.cache = cache || {};
 		this.FS_ACCURACY = 2000;
 	}
 
+	/**
+	 *
+	 * @param {TODO} compiler Webpack Compiler
```

Comment:
TODO -> Compiler

Reply:
@sokra 
TS complains: Property 'compilers' does not exist on type 'Compiler'.
Still change the type to `Compilers`?